snooty/test_postprocess.py

```diff
@@ -3484,6 +3484,30 @@ def test_collapsible_headings() -> None:
 Heading of the page
 ===================
 
+.. collapsible::
+    :heading: Collapsible heading
+    :sub_heading: Subheading
+
+    ~~~~~~~~~~~~~~~
+    This is content
+    ~~~~~~~~~~~~~~~
+""",
+        }
+    ) as result:
+        page = result.pages[FileId("index.txt")]
+
+    with make_test(
```

Comment:
Are we testing anything for this case? Should we remove? Otherwise, we can probably consolidate with this other [test case](https://github.com/mongodb/snooty-parser/pull/618/commits/6718998a14c15754262af988d3a20b520f064f3b#diff-bfac8a18bf662c13854408d2116c9293dc488fde36361670d1c7af27becdafa3R3621)

Reply:
removed this null test!